releasenotes/releasenotes_V0.8.6.md

```diff
@@ -15,12 +15,12 @@ Installing and/or upgrading to this release should be done using the `helm upgra
 
 **Installation** (using `--install`)
 ```console
-helm upgrade keptn keptn --install -n keptn --create-namespace --wait --version=0.8.6 --repo=https://storage.googleapis.com/keptn-installer
+helm upgrade keptn keptn --install -n keptn --create-namespace --wait --version=0.8.6 --repo=https://charts.keptn.sh
```

Comment:
was this change intended?

Reply:
yes, the historical charts are also online in the new charts repo